proto-wine/ftnoir_protocol_wine.h

```diff
@@ -24,8 +25,8 @@ struct settings : opts
                 esync{b, "esync", true};
 
     value<int>     proton_appid{b, "proton-appid", 0};
-    value<QString> proton_version{b, "proton-version", {} },
-                   wineprefix{b, "wineprefix", "~/.wine"};
+    value<QVariant> proton_path{b, "proton-version", {} };
```

Comment:
Is there a reason this is a QVariant? Otherwise it should be a QString.

Reply:
I tried both, only using QVariant works.  I'm trying to access the data attached to the item in the combox but when using a QString i always get the label.

The combo box labels are set to strings like `Proton 3.16` and the data field is set to `/home/russell/.steam/steam/steamapps/common/Proton 3.16`

I've been using  `tie_setting(s.proton_path, ui.proton_version);` to connect the proton_path to the combo box in both cases.

When i make the proton_path a QString
```
s.proton_path() == "Proton 3.16"
```

when i make proton_path a QVariant
```
s.proton_path().toString() == "/home/russell/.steam/steam/steamapps/common/Proton 3.16"
```

Reply:
Then the combobox's userdata stores more information than the user can select. The user-visible option is "Proton 3.16" but the persistent setting is bound to an absolute path.

But that's a PITA. We've already gone through many revisions with this PR. I'm going to merge it and work on it later.